### DISCUSSION TITLE
How to use it?

### DISCUSSION THREAD

**Discussion by naveen-malla** [JosephLai241/URS#54] [Q&A]
This might seem like a dumb question but I've tried to but it isn't working. I want to scrape all the comments in order for a given post URL.

I cloned the repo and went inside the urs folder where urs.py file is present and gave the following command and it did not work.
python Urs.py -c "https://www.reddit.com/r/germany/comments/yydfai/what_is_your_opinion_of_graffiti_all_over_walls/"

please let me know if I'm doing something wrong. Or if there is a tutorial online showing the usage of this project I'd be grateful if you can share it. 

Thanks.

<img width="1328" alt="image" src="https://user-images.githubusercontent.com/30531283/202672244-a494f1d4-2d29-41bc-aca0-171556eb6fff.png">

praw is already installed and I also created a praw.ini file and gave client_id, client_secret, username and password at the bottom of the file.

**Comment by JosephLai241** [comment]
You do not need to create a `praw.ini` file. The [Getting Started section](https://github.com/JosephLai241/URS#getting-started) in the `README` contains a link to a separate document describing what you need to do to provide credentials for `URS`.

**Reply by naveen-malla** [reply]
hey, @JosephLai241 I tried the things you said. I was getting the module not found error. I followed the same steps that were mentioned in the link. I added export PYTHONPATH="/Users/my_user/code" in the bash_profile file but when I try echo $PYTHONPATH it doesn't print anything. I checked by printing "sys.path" and I was able to see the path "/Users/naveenmalla/Desktop/URS/URS" inside it and the output is still module urs not found. I changed the path to "/Users/naveenmalla/Desktop/URS/URS/urs" and "/Users/naveenmalla/Desktop/URS/URS/urs/Urs.py" and checked with both of them. It still did not work. I also added this path is os.environment variable and no change.

Please help me what should I check. I'm stuck. I feel helpless.

I'm using the command    python Urs.py -c "URL of the reddit post" and the error doesn't change whatever I change.
<img width="357" alt="Screenshot 2022-11-25 at 5 58 45 PM" src="https://user-images.githubusercontent.com/30531283/204029667-f3a07084-d931-4e0c-9a83-884471720d68.png">

**Reply by lmnzlpz** [reply]
Hey @naveen-malla I know this is an old post, but I got it working by running "PYTHONPATH=. " followed by the command, so something like this: PYTHONPATH=. poetry run python3 urs/Urs.py -r NoSleep h 10

I also ran it from the URS directory and not the URS/urs subdirectory.
